R/commonMachineLearningClassification.R

```diff
@@ -413,7 +432,22 @@
                         distribution = "multinomial", n.cores=1) #multiple cores breaks modules in JASP, see: INTERNAL-jasp#372
```

Comment:
Not really important, but can you use a token/variable for n.cores input which  we can change very quickly once INTERNAL-jasp#372 is solved? The hope is that the input is always of th eform `n.cores=` but I doubt that that's the case across various packages.